### ISSUE TITLE
STRETCH GOAL! Get a cookie 🍪 sent to the client

### ISSUE THREAD

**Issue by ecumene** [IsabelMcIlroy/KnittingProjectManager#16]
# DEPENDS ON: #17

- [x] Add a login route
- [x] Add your cookie to the `res`
- [ ] Check that it's in your dev tools

`POST /login`
```
res.cookie('cookieName', 'cookieValue')
```
